gluster-exporter/metric_brick.go

```diff
@@ -128,19 +129,17 @@ func diskUsage(path string) (disk DiskStatus, err error) {
 	return
 }
 
-func brickUtilization() {
+func brickUtilization() error {
```

Comment:
Add the return statement  ```return nil``` at the end of the function.

Reply:
Ack